pset-iterators.js

```diff
@@ -184,6 +230,17 @@ console.log('Problem 7:')
 
 // Add your code below this line
 
+const shortTermPresidents = presidents.filter(function (name) {
```

Comment:
@jacobkelman great job -  looks like you decided to go with a more precise approach to calculating the number of years a president spent in office.  Working with time and dates in programming can be tricky. 

In solution, I used a less precise, but more simple way to calculate the years spent in office by only taking into account the year as opposed to the entire date: https://github.com/jsd20191008/hw-02-iterators-solution/blob/master/pset-iterators.js#L208